baker/GrapherBaker.tsx

```diff
@@ -212,27 +225,35 @@ export async function renderDataPageV2(
         grapher.slug
     )
 
-    // Get the charts this variable is being used in (aka "related charts")
-    // and exclude the current chart to avoid duplicates
-    datapageData.allCharts = await getRelatedChartsForVariable(
-        knex,
-        variableId,
-        grapher && "id" in grapher ? [grapher.id as number] : []
-    )
+    let imageMetadata: Record<string, ImageMetadata> = {}
+    let tagToSlugMap: Record<string, string> = {}
 
-    datapageData.relatedResearch =
-        await getRelatedResearchAndWritingForVariables(knex, [variableId])
+    // If we're baking to an archival page, then we want to skip a bunch of sections
+    // where the links would break
 
-    const relatedResearchFilenames = datapageData.relatedResearch
-        .map((r) => r.imageUrl)
-        .filter((f): f is string => !!f)
+    if (!archiveInformation) {
```

Comment:
This is what hides these sections from data pages - the relevant `datapageData` fields are not present in the first place.